### DISCUSSION TITLE
Train the model with custom dataset

### DISCUSSION THREAD

**Discussion by falahatshahrzad** [mindee/doctr#1550] [Q&A]
### 🚀 The feature

Hello,
Can you guide me how to train the doctr model for text detection and recognition using a custom dataset?
Thank you for your guidance 

### Motivation, pitch

I am working on persian and English text detection and recognition in the images.

**Comment by felixdittrich92** [comment]
Please take a look at: https://mindee.github.io/doctr/using_doctr/custom_models_training.html

:)
